src/insights_client/tests/test_metrics.py

```diff
@@ -285,3 +285,130 @@ def test_metrics_post_event_proxy(post, config_file_factory, rhsm_config_file_fa
         proxies={"https": "http://user:password@localhost:3128"},
     )
 
```

Comment:
The RawConfigParser initialization is repeated too many times. I know this doesn’t have to be a fixture, just doing so for consistency with the other factories. It may not be necessary if the _\_is_offline_ test would be made parametrized.

```suggestion

@fixture
def raw_config_parser_factory(config_file_factory):
    def factory(config):
        config_file = config_file_factory(config)
        parser = configparser.RawConfigParser()
        parser.read(config_file.name)
        return parser
    return factory
```